nexus/db-queries/src/db/datastore/vpc.rs

```diff
@@ -681,6 +686,41 @@ impl DataStore {
             .filter(service_network_interface::time_deleted.is_null())
             .select(Sled::as_select());
 
+        // ... and we also need to query for the current target blueprint to
+        // support systems that _are_ under Reconfigurator control. This query
+        // needs a join with a subquery, which isn't currently supported by
+        // diesel's DSL, so we fall back to a raw SQL string and rely on tests
+        // to ensure that this is valid.
```

Comment:
Where's the raw SQL that this refers to?

Reply:
Oops, this comment is old. I found a different way to write the query that diesel does accept; will remove.